pxt.json

```diff
@@ -4,7 +4,8 @@
     "description": "Images from Adafruit",
     "license": "MIT",
     "dependencies": {
-        "screen": "*"
+        "screen": "*",
+        "device": "file:../../libs/device"
```

Comment:
I don’t think you need this.

Reply:
This is needed to run buildsprites cli. I removed it and updated the readme to reflect that it is needed to run the cli.